### ISSUE TITLE
Error when deploying with mup

### ISSUE THREAD

**Issue by jamauro** [activitree/meteor-push#39]
Here's what I'm seeing in the `eb-engine.log`:

```
An error occurred during execution of command [app-deploy] - [StageApplication]. 
Stop running the command. 
Error: chown /var/app/staging/programs/server/npm/node_modules/meteor/activitree_push/node_modules/protobufjs/.node_modules-garbage-1d9fzmw.qib4/@types: no such file or directory 
```

I'm guessing this is on the mup side but wondering if you had any ideas what might be causing it?

**Comment by paulincai** [comment]
Could you please try to run this as a local page and let's see if we get any different outcome. In your project, you can crate a 'packages' folder in your root (e.g. app/packages). Under that create a folder names 'meteor-push' and copy from this repo the lib folder as well as the package.js file. Remove and add this package again to your project and deply.

Looking forward to hearing on this.

**Comment by jamauro** [comment]
That worked. Thanks! 

How should I set it up so that I don't need the local package going forward? Should I just remove the local package and try adding again or is there something else that you recommend?

**Comment by paulincai** [comment]
Unfortunately I do not recommend anything at the moment other than that this needs to be fixed :).
This problem was introduced with the last commit which makes it compatible with Meteor 2.3+. We'll need to understand how this dependency affects the package.

**Comment by paulincai** [comment]
@jamauro this has been fixed within 2.0.10
